fossunited/fossunited/forms.py

```diff
@@ -2,7 +2,7 @@
 
 import frappe
 
-from fossunited.doctype_ids import USER_PROFILE
+from fossunited.doctype_ids import CUSTOM_ANSWERS, USER_PROFILE
```

Comment:
:red_circle: why we immediately need to fix ruff 

```suggestion
from fossunited.doctype_ids import CUSTOM_ANSWER, USER_PROFILE
```

Reply:
could you please check the conversation on mattermost regarding ruff? I assume you missed it.